src/Citeproc/Eval.hs

```diff
@@ -2293,6 +2319,8 @@ getDisplayName nameFormat formatting order name = do
                             case formatSuffix f of
                               Just t | endsWithSpace t -> Nothing
                               _ -> Just " " } [formatted f x, y]
+      Linked lt i x <+> y =
+        formatted mempty{ formatDelimiter = Just " " } [Linked lt i x, y]
```

Comment:
does this implementation of `<+>` make sense?  It wasn't clear to me what the intended meaning is

Reply:
Yeah, that's the sort of thing one wishes the author had documented in the code.
(Even the author wishes this.)
If memory serves, these are used to format names.  `<:>` goes where a comma (or whatever the separator is) will go and `<+>` where a space will go.
IT shouldn't matter much for Linked, since this won't be used in names, but we should still try to get it right.
I think you have.